default.nix

```diff
@@ -0,0 +1,38 @@
+# Licensed under GNU Lesser General Public License v3 or later, see COPYING.
+# Copyright (c) 2019 Alexander Sosedkin and other contributors, see AUTHORS.
+
+{ }:
```

Comment:
Can you explain the reason behind this change?

Reply:
In h-m config, we import it as a function, so I made it into one.